tokio/src/io/util/read_exact.rs

```diff
@@ -11,8 +11,8 @@ use std::task::{Context, Poll};
 /// A future which can be used to easily read exactly enough bytes to fill
 /// a buffer.
 ///
-/// Created by the [`AsyncReadExt::read_exact`][read_exact].
-/// [read_exact]: [crate::io::AsyncReadExt::read_exact]
+/// Created by the [`read_exact`].
+/// [`read_exact`]: crate::io::AsyncReadExt::read_exact
```

Comment:
This just makes the comment harder to understand. We explicitly write out `AsyncReadExt` because there are multiple methods of the same name.